keyboards/ergodash/rev1/keymaps/totuus1157/keymap.c

```diff
@@ -0,0 +1,177 @@
+  ),
+
+  /* Adjust
+   * ,----------------------------------------------------------------------------------------------------------------------.
+   * |      |      |      |      |      |      |      |                    |      |      |      |      |      |      |      |
+   * |------+------+------+------+------+------+---------------------------+------+------+------+------+------+------+------|
+   * |      | Reset|RGB ON|  MODE|  HUE-|  HUE+|      |                    |      |  SAT-|  SAT+|  VAL-|  VAL+|      |      |
+   * |------+------+------+------+------+------+---------------------------+------+------+------+------+------+------+------|
+   * |      |      | BL ON|  BRTG|  INC|   DEC|      |                    |      |      |      |      |      |      |      |
+   * |------+------+------+------+------+------+---------------------------+------+------+------+------+------+------+------|
+   * |      |      |      |      |      |      |      |                    |      |      |      |      |      |      |      |
+   * |-------------+------+------+------+------+------+------+------+------+------+------+------+------+------+-------------|
+   * |      |      |      |      ||||||||      |      |      ||||||||      |      |      ||||||||      |      |      |      |
+   * ,----------------------------------------------------------------------------------------------------------------------.
+   */
+  [_ADJUST] = LAYOUT(
+    _______, _______, _______, _______, _______, _______,_______,                       _______, _______, _______, _______, _______, _______, _______, \
+    _______, RESET  , RGB_TOG, RGB_MOD, RGB_HUD, RGB_HUI,_______,                       _______, RGB_SAD, RGB_SAI, RGB_VAD, RGB_VAI, _______, _______, \
+    _______, _______, BL_TOGG, BL_BRTG, BL_INC , BL_DEC ,_______,                       _______, _______, _______, _______, _______, _______, _______, \
+    _______, _______, _______, _______, _______, _______,_______,                       _______, _______, _______, _______, _______, _______, _______, \
+    _______, _______, _______, _______,          _______,_______,_______,       _______,_______, _______,          _______, _______, _______, _______  \
+  )
+};
+
+const uint16_t PROGMEM we_combo[] = {KC_W, KC_E, COMBO_END};
+const uint16_t PROGMEM io_combo[] = {KC_I, KC_O, COMBO_END};
+const uint16_t PROGMEM er_combo[] = {KC_E, KC_R, COMBO_END};
+const uint16_t PROGMEM ui_combo[] = {KC_U, KC_I, COMBO_END};
+
+combo_t key_combos[COMBO_COUNT] = {
+  [ER_LBRACKET] = COMBO(er_combo, KC_LBRC),
+  [UI_RBRACKET] = COMBO(ui_combo, KC_RBRC),
+  [WE_MINUS] = COMBO(we_combo, KC_MINS),
+  [IO_EQUAL] = COMBO(io_combo, KC_EQL)
+};
+
+#ifdef AUDIO_ENABLE
+float tone_qwerty[][2]     = SONG(QWERTY_SOUND);
+#endif
+
+void persistent_default_layer_set(uint16_t default_layer) {
+  eeconfig_update_default_layer(default_layer);
+  default_layer_set(default_layer);
+}
```

Comment:
Could you remove this function?  

It's been deprecated in favor of the `set_single_persistent_default_layer` function (which is a core function).

https://docs.qmk.fm/#/ref_functions?id=setting-the-persistent-default-layer
```suggestion
```